### DISCUSSION TITLE
PybNano V2 firmware break down

### DISCUSSION THREAD

**Discussion by IHOXOHI** [micropython/micropython#12504] [STM32 / Pyboard]
Hi,
I updated a new pybnano V2 with the only one firmware existing for V1.1 version of pybnano...
Now, nothing happen... I can press buttons and the LEDs toggle but no dfu boot, no rshell, ..., nothing.
Thanks for your help.

**Comment by rkompass** [comment]
This board has a stm32F411 on it. So you should be able to use a firmware for WeAct Blackpill board like [here](https://github.com/mcauser/WEACT_F411CEU6/issues/10).

If this works and you have problems to compile the up-to-date MP version (1.20....) I may help.

**Reply by rkompass** [reply]
"Official" Firmware is [here](https://github.com/micropython-Chinese-Community/MicroPython_firmware/tree/master/PYBNanoV2).

**Comment by IHOXOHI** [comment]
Thanks a lot,
I will try to compile my first micropython on board, from my raspberry pi 3A+, at 12V solar... If I have understood correctly... If I succeed for sure I will ask you how can I update; I need to use uasyncio.
Thanks.
I

**Comment by IHOXOHI** [comment]
Hi,
I have updated another pybnano with the right firmware in .dfu... I knew the usage of a .dfu which is easy. This board is very fast! And looks stable as well... Now I can use uasyncio with this jewel... Thanks.
For build a .bin, it's a little bit more complex for me... I haven't a ttyACM0, so I suppose that I have to use a uart connection? Or a spi? What module could be use for this type of connection, ..., I will find, but it will take time...
Thanks.

**Comment by IHOXOHI** [comment]
I haven't found... It seems that it couldn't be possible... Better take care to upgrade the good firmware, before!
